### PR TITLE
[FEAT] add navigation diagram example

### PR DESCRIPTION
**Pull request by tbouffard**
**Important**
* as the npm package version providing navigation feature is not publicly available, I built it locally from https://github.com/process-analytics/bpmn-visualization-js/pull/774 and stored the generated javascript directly in the code
* when bpmn-visualization@0.5.0 is available, please remove the custom javascript file and load the package from a CDN

[__:fast_forward: Live demo while this PR is opened__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/feat/add_diagram_navigation_example/examples/index.html)
[__:fast_forward: Live demo prior review__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/3579e55/examples/index.html)

### CODE REVIEW

File: examples/diagram-navigation/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# BPMN Diagram navigation
+
+Javascript example to demonstrate how to navigate the BPMN diagram with the mouse..
+- [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/01-getting-started/index.html)
```
Comment (csouchet) [inline examples/diagram-navigation/README.md:4]: I think it's not the good link for the demo:
```suggestion
- [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/diagram-navigation/index.html)
```

File: examples/diagram-navigation/js/bpmn-visualization.js [hunk 2]
```diff
@@ -0,0 +1 @@
...,mxCompactTreeLayout.prototype.parentsChanged=null,mxCompactTreeLayout.prototype.moveTree=!1,mxCompactTreeLayout.prototype.visited=null,mxCompactTreeLayout.prototype.levelDistance=10,mxCompactTreeLayout.prototype.nodeDistance=20,mxCompactTreeLayout.prototype.resetEdges=!0,mxCompactTreeLayout.prototype.prefHozEdgeSep=5,mxCompactTreeLayout.prototype.prefVertEdgeOff=4,mxCompactTreeLayout.prototype.minEdgeJetty=8,mxCompactTreeLayout.prototype.channelBuffer=4,mxCompactTreeLayout.prototype.edgeRouting=!0,mxCompactTreeLayout.prototype.sortEdges=!1,mxCompactTreeLayout.prototype.alignRanks=!1,mxCompactTreeLayout.prototype.maxRankHeight=null,mxCompactTreeLayout.prototype.root=null,mxCompactTreeLayout.prototype.node=null,mxCompactTreeLayout.prototype.isVertexIgnored=function(t){return mxGraphLayout.prototype.isVertexIgnored.apply(this,arguments)||0==this.graph.getConnections(t).length},mxCompactTreeLayout.prototype.isHorizontal=function(){return this.horizontal},mxCompactTreeLayout.prototype.execute=function(t,e){this.parent=t;var n=this.graph.getModel();if(null==e)if(0<this.graph.getEdges(t,n.getParent(t),this.invert,!this.invert,!1).length)this.root=t;else{if(0<(o=this.graph.findTreeRoots(t
```
Comment (csouchet) [inline examples/diagram-navigation/js/bpmn-visualization.js:1]: I think we can delete this and replace by the 0.5.0 tag of BPMN Visualiztion.

**Comment by csouchet** [comment]
The panning & the zoom work very well 👍

**Review by aibcmars — APPROVED** [review]
LGTM

**Pull request merged**